assets/prefabs/items/WoodenBucket.prefab

```diff
@@ -9,6 +9,7 @@
   },
   "FluidContainerItem": {
     "maxVolume": 1000,
+    "fillingAmount": 100,
```

Comment:
For game play reasons I would not make it too annoying for the player to fill a bucket. 
```suggestion
    "fillingAmount": 500,
```